pkgs/development/compilers/dotnet/buildDotnet.nix

```diff
@@ -13,10 +13,14 @@ assert builtins.elem type [ "aspnetcore" "netcore" "sdk"];
 , curl
 }: 
 let pname = if type == "aspnetcore" then "aspnetcore-runtime" else if type == "netcore" then "dotnet-runtime" else "dotnet-sdk";
+    suffix = {
+      x86_64-linux  = "x86";
```

Comment:
Typo?

```suggestion
      x86_64-linux  = "x64";
```

Reply:
Yes! Thanks, sorry to have missed that. I will amend and squash.